src/applications/facility-locator/utils/helpers.js

```diff
@@ -216,3 +219,39 @@ export const isVADomain = website => {
   const regExp1 = /https?:\/\/(?:www\.|staging\.)?va\.gov(\/*)/;
   return regExp1.test(website);
 };
+
+const headingStyle = {
+  fontWeight: '700',
+  fontFamily: 'Bitter, Georgia, Cambria, Times New Roman, Times, serif',
+  lineHeight: '1.3',
+  clear: 'both',
+};
+
+const ddStyle = {
+  margin: '2rem 0 .5rem 0',
+  lineHeight: '1.5',
+};
```

Comment:
Why can't we put these in src/applications/facility-locator/sass/facility-locator.scss?

Reply:
used class names instead